From 854505c4c2363c86ff0717aa9d4e83a274784f46 Mon Sep 17 00:00:00 2001
From: Michael Stahl <mstahl@redhat.com>
Date: Fri, 5 Dec 2014 23:52:34 +0100
Subject: [PATCH 2/5] fdo#72811: Revert "Broken security/password when saving
 to older odf (sx?)

... formats."

This exports ODF 1.1 files as ODF 1.2 packages, which is quite wrong.
Since we don't support OOoXML export any more, just revert it.

This reverts commit 06b918053fb9cf455cf431ca001a63030092d018.

(cherry picked from commit ebfec3517d001f8aa8baaabde7c4af6b01347b95)

Change-Id: I27bdd83f96e46be054f808ff6782ad16ecc90da5
Reviewed-on: https://gerrit.libreoffice.org/13325
Reviewed-by: Markus Mohrhard <markus.mohrhard@googlemail.com>
Tested-by: Markus Mohrhard <markus.mohrhard@googlemail.com>
(cherry picked from commit 5faedcab93d2f987f91e2750119198b18891f4d1)
---
 sfx2/source/doc/objstor.cxx | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/sfx2/source/doc/objstor.cxx b/sfx2/source/doc/objstor.cxx
index df12f4f..a1eb2b7 100644
--- a/sfx2/source/doc/objstor.cxx
+++ b/sfx2/source/doc/objstor.cxx
@@ -382,7 +382,7 @@ void SfxObjectShell::SetupStorage( const uno::Reference< embed::XStorage >& xSto
                 aEncryptionAlgs[1].Value <<= xml::crypto::CipherID::BLOWFISH_CFB_8;
                 aEncryptionAlgs[2].Value <<= xml::crypto::DigestID::SHA1_1K;
 
-                if ( nVersion >= SOFFICE_FILEFORMAT_8 )
+                if ( nDefVersion >= SvtSaveOptions::ODFVER_012 )
                 {
                     try
                     {
-- 
1.9.3

